File: src/main/java/com/rs/java/game/area/multi/MorytaniaMulti.java
package com.rs.java.game.area.multi;

import com.rs.java.game.area.Area;

public final class MorytaniaMulti extends Area {

	@Override
	public Area update() {
		return this;
	}

	@Override
	public String name() {
		return "Morytania Multi";
	}

	public MorytaniaMulti() {
		super(13876);
	}

	@Override
	public boolean member() {
		return false;
	}

	@Override
	public Environment environment() {
		return Environment.MULTI;
	}

}

File: src/main/java/com/rs/kotlin/game/world/area/areas/multi/MorytaniaMulti.kt
package com.rs.kotlin.game.world.area.areas.multi

import com.rs.kotlin.game.world.area.Area

class MorytaniaMulti : Area(13876) {
    override fun update(): Area {
        return this
    }

    override fun name(): String {
        return "Morytania Multi"
    }

    override fun member(): Boolean {
        return false
    }

    override fun environment(): Environment {
        return Environment.MULTI
    }
}
